force-app/main/default/lwc/geGift/geGift.js

```diff
@@ -160,14 +187,19 @@ class Gift {
             fields: { ...this._fields },
             softCredits: JSON.stringify([ ...this._softCredits.unprocessedSoftCredits() ]),
             processedSoftCredits: JSON.stringify([ ...this._softCredits.processedSoftCredits() ]),
-            schedule: { ...this._schedule }
+            schedule: { ...this._schedule },
+            hasConvertedToRecurringBatchItemType: this._hasConvertedToRecurringBatchItemType,
+            hasConvertedToElevateOneTimeBatchItemType: this._hasConvertedToOneTimeBatchItemType
         }
     }
 
-    isAuthorized() {
-        return (this.getFieldValue(PAYMENT_STATUS.fieldApiName) === PAYMENT_STATUSES.AUTHORIZED
-            || this.getFieldValue(PAYMENT_STATUS.fieldApiName) === PAYMENT_STATUSES.PENDING)
-            && !!this.getFieldValue(PAYMENT_ELEVATE_ID.fieldApiName);
+    hasElevateRemovableStatus() {
+        return this.getFieldValue(STATUS.fieldApiName) !== GIFT_STATUSES.IMPORTED && (
+                this.getFieldValue(PAYMENT_ELEVATE_ID.fieldApiName) &&
+                this.getFieldValue(PAYMENT_STATUS.fieldApiName) === PAYMENT_STATUSES.AUTHORIZED ||
+                this.getFieldValue(PAYMENT_STATUS.fieldApiName) === PAYMENT_STATUSES.PENDING
+            ) ||
+            this.hasCommitmentId();
```

Comment:
Wouldn't this last condition allow an imported RD to be removed from the Batch, and is that ok? I only ask because we explicitly check for a Status of **not** Imported in the first condition.

Reply:
@lparrott You're right. Technically, I don't think it would be a problem since it's already been processed at that point and the Update button is disabled for Imported but I updated the parentheses to be more accurate